### ISSUE TITLE
Injected extension name

### ISSUE THREAD

**Issue by youPickItUp** [novasamatech/nova-wallet-dapp-js#14]
I think it would be more appropriate if you could use different name than original polkadot-js extension:
https://github.com/novasamatech/nova-wallet-dapp-js/blob/eb269ad99be14736c3ff3bad85ab035d6b073993/src/index.ts#L9
In case of Nightly Connect integration we are currently using, it shows your extension as polkadot-js with their icon and name.

Disclaimer -- Even though I'm not aware of any issues such change could bring it could be possible that some site allows only `polkadot-js` integration. If that's the case... well :hankey:

**Comment by youPickItUp** [comment]
@leohar, sorry to bother you. I just didn't want the issue to be left unnoticed :see_no_evil:

**Comment by laurogripa** [comment]
Agreed. There's probably a good explanation for that, but I don't understand why the extension name is `polkadot-js`. I had to implement a weird edge case workaround, treating Nova's extension name as "polkadot-js", and using mobile detection to differentiate between mobile & desktop browsers to decide which "polkadot-js" to show.

See here: 

It's also the case for Hydra's UI, see here:
https://github.com/galacticcouncil/hydration-ui/blob/master/src/sections/web3-connect/wallets/NovaWallet.ts

And here:
https://github.com/galacticcouncil/hydration-ui/blob/ede41140bb69cc0aea9a22ccdda4d39446a3c6d8/src/sections/web3-connect/constants/providers.ts

It sounds like a very simple fix, we just need to be aware that changing the extension name might break a lot (if not all) UIs that support Nova in the ecosystem. I think it needs to be a coordinated fix.

**Comment by ERussel** [comment]
Hey everyone! Thank you for raising the issue. We are planing to add a separate extension name for the Nova Wallet. I will right in the issue once it is released.
